core/storage/exploration/gae_models.py

```diff
@@ -317,6 +317,9 @@ class ExpSummaryModel(base_models.BaseModel):
     # Aggregate user-assigned ratings of the exploration
     ratings = ndb.JsonProperty(default=None, indexed=False)
 
+    # Scaled average rating for the exploration.
+    scaled_average_rating = ndb.FloatProperty()
```

Comment:
Oh, this needs to be (indexed=True) for the query to work, I _think_

Also shouldn't there be an additional stanza in index.yaml? Did you test this on a local dev server? It should autogenerate an additional stanza when a query that hits get_top_rated() is made.